<?xml version="1.5"?>

<launch>
  <arg name="marker_name" default="aruco_single_0"/>
  <arg name="world_name" default="base_link"/>

  <node
    pkg = "tgrip_cv_marker_tf"
    type = "broadcaster_node"
    name = "aruco_tf_broadcaster_$(arg marker_name)"
    output = "screen">
    <param name = "marker_name" value = "$(arg marker_name)"/>
    <param name = "world_name" value = "$(arg world_name)" />
  </node>

</launch>
